index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="flex-container">
+           <h1 class="flex-item">Welcome</h1>
```

Comment:
- [ ] Kindly use `a` tag for the logo for three reasons: convention, semantics and user experience.

Reply:
I added the `a` tag to the logo section.